ClemBot.Bot/bot/cogs/info_cog.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import discord
+import discord.ext.commands as commands
+import bot.extensions as ext
+from bot.consts import Colors, Claims
+from bot.messaging.events import Events
+
+log = logging.getLogger(__name__)
+
+class InfoCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @ext.command()
+    @ext.required_claims(Claims.moderation_infraction_view)
+    @ext.ignore_claims_pre_invoke()
+    async def info(self, ctx, user: discord.User = None):
+        #If the command is invoked without a specified user, it will return info on the calling user
+        if not user:
+            user = ctx.author
+        
+        log.info(f'User {ctx.author} has ran info command on user {user.name}')
+
+        embed = discord.Embed(title = f'Guild Member Information', color=Colors.ClemsonOrange)
+        embed.set_author(name=f'{user.name}#{user.discriminator}', icon_url=user.avatar_url)
+
+        user_info = f'» **Nickname:** {user.mention}'
+        user_info += f'\n» **ID:** {user.id}'
+        #Some info doesn't need to be available to anyone to check on for other users
+        claims = await self.bot.claim_route.get_claims_user(ctx.author)
+        if (await self.bot.claims_check(ctx=ctx)) or (ctx.author.id == user.id): 
+            #prints the datetime in the format of <Months> <day> <year> <hours>:<minutes>:<seconds> <AM/PM>
+            user_info += '\n» **Created:** ' + user.created_at.strftime('%b %d %Y %I:%M:%S %p')
+        embed.add_field(name='**User ID:**', value=user_info)
+        embed.set_thumbnail(url=user.avatar_url)
+
+        #since member objects include guild ID's and information, we try to get a member object for the target user
+        member = ctx.guild.get_member(user.id)
+        #check to see if the target user is in the calling context's guild. If None then the user isn't in the calling server
+        if member:
+
+            # We don't want moderation info available for regular guild users to check on others
+            if (await self.bot.claims_check(ctx=ctx)): 
```

Comment:
Redundant parentheses 
```suggestion
            if await self.bot.claims_check(ctx=ctx): 
```